.github/workflows/auto-update.yml

```diff
@@ -0,0 +1,81 @@
+# Modifed from https://github.com/flathub/net.veloren.veloren/raw/master/net.veloren.veloren.yaml
+
+name: Auto update
+
+env:
+  BUILDER_TOOLS_URL: https://github.com/proletarius101/flatpak-builder-tools.git
```

Comment:
Any reason not to point at https://github.com/flatpak/flatpak-builder-tools.git here?